backends/advanced/src/advanced_omi_backend/chat_service.py

```
@@ -142,6 +143,33 @@ def __init__(self):
+            str: System prompt for chat interactions
+        """
+        try:
+            reg = get_models_registry()
+            if reg and hasattr(reg, 'chat'):
+                chat_config = reg.chat
+                prompt = chat_config.get('system_prompt')
+                if prompt:
+                    logger.info(f"✅ Loaded chat system prompt from config (length: {len(prompt)} chars)")
+                    logger.debug(f"System prompt: {prompt[:100]}...")
+                    return prompt
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)